src/cli/commands/lint.ts

```diff
@@ -50,6 +50,10 @@ linting ./openapi.yaml
       char: 'v',
       description: 'increase verbosity',
     }),
+    skip: flagHelpers.string({
```

Comment:
Could we rename this to —skip-rule= perhaps to make it a bit more clear, and to leave room to skip other things in the future?

Reply:
Shall we follow the same convention in the name, like maxResults ? skipRules : skip-rules 😄

Reply:
Perfect.

Reply:
I am wondering, we are adding `--ruleset=foo.yaml --ruleset=bar.yaml` in another PR, which is singular and allows multiple. This `--skip-rules` is plural and supports `,`, and this seems inconsistent. 

Can you name it `--skip-rule` (keep on supporting the `,`) and I will make sure it works with multiple calls to make it all consistent.

Reply:
changed skip to skipRule